packages/back-end/src/integrations/Mssql.ts

```diff
@@ -78,53 +67,14 @@ export default class Mssql extends SqlIntegration {
   formatDateTimeString(col: string): string {
     return `CONVERT(VARCHAR(25), ${col}, 121)`;
   }
-  async getInformationSchema(): Promise<InformationSchema[]> {
-    const databaseName = this.params.database;
-
-    if (!databaseName)
+  getInformationSchemaFromClause(): string {
+    if (!this.params.database)
       throw new MissingDatasourceParamsError(
         "To view the information schema for a MS Sql dataset, you must define a default database. Please add a default database by editing the datasource's connection settings."
       );
-
-    const queryString = `SELECT
-    table_name,
-    table_catalog,
-    table_schema,
-    COUNT(column_name) as column_count
-  FROM
-    ${databaseName}.INFORMATION_SCHEMA.COLUMNS
-    GROUP BY table_name, table_schema, table_catalog`;
-
-    const results = await this.runQuery(queryString);
-
-    if (!results.length) {
-      throw new Error(`No tables found for database "${databaseName}".`);
-    }
-
-    return formatInformationSchema(results, "mssql");
+    return `${this.params.database}.information_schema.columns`;
```

Comment:
Changed this to lower-case and tested - it appears as those MS Sql is case-insensitive by default.